modules/orchestrator/proc-install-rhdh-with-orchestrator-helm-webui.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-install-rhdh-with-orchestrator-helm-webui_{context}"]
+= Install {product} ({product-very-short}) using {ocp-short} web console
```

Comment:
```suggestion
= Install {product} ({product-very-short}) using Helm from the {ocp-short} web console
```

Maybe we could clarify that this is still a Helm-based installation method from the OCP web console. Because Operators can also be installed from the OCP web console.